compose/compose-file/compose-versioning.md

```diff
@@ -252,17 +252,31 @@ Designed to be cross-compatible between Compose and the Docker Engine's
 [swarm mode](/engine/swarm/index.md), version 3 removes several options and adds
 several more.
 
+Version 3 files are supported by **Compose 1.10.0+** and require a Docker Engine
+of version **1.13.0+**.
+
 - Removed: `volume_driver`, `volumes_from`, `cpu_shares`, `cpu_quota`,
 `cpuset`, `mem_limit`, `memswap_limit`, `extends`, `group_add`. See
 the [upgrading](#upgrading) guide for how to migrate away from these.
 (For more information on `extends`, see [Extending services](/compose/extends.md#extending-services).)
 
 - Added: [deploy](/compose/compose-file/index.md#deploy)
 
+### Version 3.1
+
+Version 3.1 files are supported by **Compose 1.11.0+** and require a Docker Engine
```

Comment:
The `and require a Docker Engine` may be ambiguous; to use this version of the compose-file specification, you need either;

- Docker Compose 1.11.0 (or higher)
- Docker Engine 1.13.1 (or higher); if you want to use it for `docker stack deploy`


There may be restrictions to the engine version that is supported by each Docker Compose version though (for example, Docker Compose 1.17 requires Docker Engine 17.06 or up; https://github.com/docker/compose/blob/master/CHANGELOG.md#1170-2017-11-02)